lib/agent/reporter.rb

```diff
@@ -46,6 +64,7 @@ def create_reporter_thread
             send_report_batch unless @event_cache.num_events.zero?
           end
         end
+        report_thread.priority = 255
```

Comment:
I'm not sure we should be making ourselves more important at the expense of starving application CPU, we aren't the most important thing in the process 🤔